client/src/app/gateways/actions/action.service.ts

```diff
@@ -49,10 +50,17 @@ export class ActionService {
         return null;
     }
 
+    /**
+     * @deprecated this does not offer the handle_separately route option, which is vital for certain types of bulk requests, it's better to use `createFromArray` instead.
+     */
     public create<T>(...requests: ActionRequest[]): Action<T> {
```

Comment:
Can't you just add the optional parameter to this method? I don't see the use of the new `createFromArray` method.

Reply:
Ah, I see. I can't estimate the time to fix this, but if it's too much, I'm fine with the way it is.